### DISCUSSION TITLE
I can add repositories of pc?

### DISCUSSION THREAD

**Discussion by Nnico0w0** [termux/termux-app#3316] [Q&A] [answered]
if i add some repository of debian or Ubuntu, something can break in my sistem or termux?

**Comment by agnostic-apollo** [comment, marked as answer]
Termux repo packages are compiled specifically for termux and android. Linux distro packages are mainly not going to work and can break termux depending on package.

https://wiki.termux.com/wiki/Differences_from_Linux
